profiles/base-empty.yaml

```diff
@@ -45,4 +45,5 @@ empty_values:
   hunting_buddies: []
   empty_hunting_room_messages: []
   classes_to_teach: []
-  whirlwind_trainables: []
\ No newline at end of file
+  whirlwind_trainables: []
+  hide_type: hide
```

Comment:
I found some sublime settings to fix those trailing white spaces and endlines! hopefully gone forever.